packages/theming/atlas/package.json

```diff
@@ -13,11 +13,14 @@
     "build": "node ./scripts/build.js",
     "start": "node ./scripts/build.js start",
     "pretest:e2e:dev": "node ./scripts/build.js",
+    "pretest:e2e:collect": "node ./scripts/atlasCollectPages.js",
```

Comment:
It needs to be renamed to `pretest:e2e` in order to be executed before `test:e2e` adding `:collect` will prevent it.

Reply:
As discussed I think we can keep it separate because today we're not really comparing the screenshot when we have new pages, so it's not necessary to execute this script in the pretest as we're doing previously.